internal/presentation/presentation.go

```diff
@@ -571,8 +604,8 @@ func prettyExplanation(w io.Writer, explanation []*topdown.Event, opts topdown.P
 
 func prettyCoverage(w io.Writer, report *cover.Report) error {
 	table := tablewriter.NewWriter(w)
-	table.Append([]string{"Overall Coverage", fmt.Sprintf("%.02f", report.Coverage)})
-	table.Render()
+	_ = table.Append([]string{"Overall Coverage", fmt.Sprintf("%.02f", report.Coverage)})
+	_ = table.Render()
```

Comment:
Here we don't care about the errors? We know they won't happen?

Reply:
Errors should only happen on a bad writer... I think old tablewriter just panicked on a bad writer.
I can definitely do error handling here and bubble it up if you want.